src/main/scala/com/fulcrumgenomics/vcf/filtration/EndRepairArtifactLikelihoodFilter.scala

```diff
@@ -228,22 +228,16 @@ class EndRepairArtifactLikelihoodFilter(val distance: Int = 2,
     if (rec.getReadPairedFlag && !rec.getReadUnmappedFlag && !rec.getMateUnmappedFlag &&
       rec.getReferenceIndex == rec.getMateReferenceIndex && rec.getInferredInsertSize != 0 &&
       SamPairUtil.getPairOrientation(rec) == PairOrientation.FR) {
-
+      
       val isize      = rec.getInferredInsertSize
       val thisEnd    = if (rec.getReadNegativeStrandFlag) rec.getAlignmentEnd else rec.getAlignmentStart
       val adjustment = if (rec.getInferredInsertSize < 0) 1 else -1
       val otherEnd   = thisEnd + rec.getInferredInsertSize + adjustment
+      val (start, end) = (min(thisEnd, otherEnd), max(thisEnd,otherEnd))
```

Comment:
I wanted to, but it's important here to know which of the two ends of the insert are which, which I kinda lose with the shiny new method.